stores/amazon.py

```diff
@@ -446,7 +343,7 @@ def check_if_captcha(self, func, args):
                 )
                 self.take_screenshot("title-fail")
                 time.sleep(60)
-                self.driver.close()
```

Comment:
why are we closing on captcha fail vs returning back to checking?

Reply:
not sure why we were closing. adjusted so we don't close and go back to checking now. Unless you think we should close?

Reply:
yeah I dont see a reason for it, so leaving it commented out is probably for the best until we find a reason we should be doing it.